Saqo/databases/13-05-24/homework.js

```diff
@@ -0,0 +1,141 @@
+// Products
+
+db.Products.drop()
+db.createCollection("Products")
```

Comment:
for the best practice, you can create new collections with specific options - https://www.mongodb.com/docs/manual/reference/method/db.createCollection/